src/common/contracts/core/campaigns/client.ts

```
@@ -21,54 +21,45 @@ const contractApi = naxiosInstance.contractApi({
+export const update_campaign = ({ args }: UpdateCampaignParams) =>
+  contractApi.call<UpdateCampaignParams["args"], Campaign>("update_campaign", {
     args,
     deposit: floatToYoctoNear(0.021),
     gas: FULL_TGAS,
   });
-};
 
-export const delete_campaign = ({ args }: { args: { campaign_id: CampaignId } }) => {
-  return contractApi.call<{}, void>("delete_campaign", {
+export const delete_campaign = ({ args }: { args: { campaign_id: CampaignId } }) =>
+  contractApi.call<{}, void>("delete_campaign", {
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)